tests/FakeItEasy.Analyzer.Tests/UnusedCallSpecificationAnalyzerTests.cs

```diff
@@ -97,17 +98,115 @@ interface IFoo { int Bar(); }
                 test,
                 new DiagnosticResult
                 {
-                    Id = UnusedCallSpecificationAnalyzer.DiagnosticId,
+                    Id = DiagnosticDefinitions.UnusedCallSpecification.Id,
                     Message =
-                        "Unused call specification for 'foo.Bar()'; did you forget to configure or assert the call?",
+                        "Unused call specification for 'A.CallTo(() => foo.Bar())'; did you forget to configure or assert the call?",

```

Comment:
Ah, yes. I noticed it and intended to fix it, but I was focused on something else and it slipped my mind.